README.md

```diff
@@ -2,3 +2,6 @@
 
 Library for one API for many git forges. (e.g. github, gitlab, pagure).
 
+# Contribution notes
```

Comment:
This section would use some more love but that's probably out of scope of this PR.